parsec/core/gui/remanence_management_widget.py

```diff
@@ -19,6 +19,16 @@
 from parsec.core.logged_core import LoggedCore
 
 
+def normalize_progress(value: int, total: int, maxi: int = 0x7FFFFFFF) -> tuple[int, int]:
+    if total <= maxi:
+        return value, total
+    assert total != 0
+    normalized_total = min(total, maxi)
+    factor = normalized_total / total
+    normalized_value = int(round(value * factor))
+    return normalized_value, normalized_total
```

Comment:
You shouldn't need `normalized_total` here because of the check in line 23 (`total` is always greater than `maxi` here) 
```suggestion
    factor = maxi / total
    normalized_value = int(round(value * factor))
    return normalized_value, maxi
```